test/widgets/app_test.dart

```diff
@@ -162,6 +159,45 @@ void main() {
         ..bottom.isLessThan(2 / 3 * screenHeight);
     });
 
+    testWidgets('choosing an account clears the navigator stack', (tester) async {
+      addTearDown(testBinding.reset);
+      await testBinding.globalStore.add(eg.selfAccount, eg.initialSnapshot());
+      await testBinding.globalStore.add(eg.otherAccount, eg.initialSnapshot());
+
+      final pushedRoutes = <Route<dynamic>>[];
+      final poppedRoutes = <Route<dynamic>>[];
```

Comment:
nit: avoid `dynamic`:
```suggestion
      final pushedRoutes = <Route<void>>[];
      final poppedRoutes = <Route<void>>[];
```